src/Workspaces/Core/Portable/Formatting/FormattingOptions.cs

```diff
@@ -99,6 +99,9 @@ private static string GetEndOfLineEditorConfigString(string option)
 
         internal static Option<bool> AllowDisjointSpanMerging { get; } = CreateOption(OptionGroup.Default, nameof(AllowDisjointSpanMerging), defaultValue: false);
 
+        internal static readonly PerLanguageOption<bool> AutoFormattingOnReturn = CreatePerLanguageOption(OptionGroup.Default, nameof(AutoFormattingOnReturn), defaultValue: true,
```

Comment:
can we move all other formatting related options if we are going to move? not sure why only this option has special meaning to be here?